### PR TITLE
Delubrum Savage Translations

### PR DESCRIPTION
**Pull request by Akurosia**

### CODE REVIEW

File: ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js [hunk 1]
```diff
@@ -235,8 +298,601 @@ export default {
     },
     {
       id: 'DelubrumSav Lord 1111-Tonze Swing',
-      netRegex: NetRegexes.startsUsing({ source: '1111-Tonze Swing', id: '57D8', capture: false }),
+      netRegex: NetRegexes.startsUsing({ source: 'Stygimoloch Lord', id: '57D8', capture: false }),
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js:301]: What happened here that you changed english regex? Is it wrong?

**Review by MaikoTan — APPROVED** [review]

File: ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js (hunk 1 above)
Comment (Legends0) [reply ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js:301]: 1111-Tonze Swing is from Stygimoloch Lord, so the English regex is wrong.

Comment (MaikoTan) [reply ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js:301]: Oh, that's the skill name!

Comment (quisquous) [reply ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js:301]: Whoops, thank you.  I was tired.

**Pull request merged**